src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -34,34 +31,32 @@
 (defn animated-bar-style [margin-value width color]
   {:position         :absolute
    :width            width
-   :transform        [{:translateX
-                       (animation/interpolate
-                        margin-value
-                        {:inputRange  [0 1]
-                         :outputRange [0 width]})}]
+   :transform        [{:translateX (animation/interpolate
+                                    margin-value
+                                    {:inputRange  [0 1]
+                                     :outputRange [0 width]})}]
    :height           3
    :background-color color})
 
 (views/defview loading-indicator [parent-width]
   (views/letsubs [blue-bar-left-margin  (animation/create-value 0)
                   white-bar-left-margin (animation/create-value 0)]
-    {:component-did-mount
-     (fn [_]
-       (animation/start
-        (animation/anim-loop
-         (animation/anim-sequence
-          [(animation/parallel
-            [(animation/timing blue-bar-left-margin (easing :in 0.19))
-             (animation/timing white-bar-left-margin (easing :in 0.65))])
-           (animation/parallel
-            [(animation/timing blue-bar-left-margin (easing :out 0.85))
-             (animation/timing white-bar-left-margin (easing :out 0.85))])
-           (animation/parallel
-            [(animation/timing blue-bar-left-margin (easing :in 0.19))
-             (animation/timing white-bar-left-margin (easing :in 0.65))])
-           (animation/parallel
-            [(animation/timing blue-bar-left-margin (easing :out 0))
-             (animation/timing white-bar-left-margin (easing :out 0))])]))))}
+    {:component-did-mount (fn [_]
```

Comment:
is this formatting also made by a tool?

Reply:
Yes, it is formatted by `cljfmt` using the config found here: https://github.com/status-im/status-react/blob/de4e73a2566abd647f97d27dd195404be379b9bf/project.clj#L32

Reply:
Fixed.